packages/components/src/components/editor/browser/browser.tsx

```diff
@@ -132,9 +132,16 @@ const BrowsersView = ({ browsers, apply, applyBtn }: BrowsersViewProps) => {
   const applyHandler = (doubleClickRow?: Row<BrowserNode>) => {
     const row = doubleClickRow ?? selectedRow();
     const browser = browsers.find(b => b.name === tab);
-    if (!row || !browser) {
+    if (!browser) {
       return; // nothing selected
     }
+    if (browser.applyModifier !== undefined && !row) {
+      apply(browser.name, browser.applyModifier());
+      return;
+    }
+    if (!row) {
+      return;
+    }
```

Comment:
instead to check the row twice I would do it like this:
```suggestion
    if (!row) {
      if (browser.applyModifier !== undefined) {
        apply(browser.name, browser.applyModifier());
      }
      return;
    }
```